docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,255 @@
+:orphan:
+
+.. doctest-skip-all
+
+******************************
+Defining Observing Constraints
+******************************
+
+Contents
+========
+
+* :ref:`constraints-built_in_constraints`
+* :ref:`constraints-user_defined_constraints`
+
+.. _constraints-built_in_constraints:
+
+Built-In Constraints
```

Comment:
(in the second option I mean have this first paragraph be "introduction to constraints" and then have "built-in constraints" be the rest, although maybe with a new lead sentence that says "first we'll talk about using constraints built into astroplan.  Later we'll show you how to define your own" or similar.